src/components/dataset/pending-submissions.vue

```diff
@@ -91,17 +91,17 @@ const convert = ref(null);
     "explanation": {
       "userAction": "You are setting Entity creation to occur when Submissions are first received by Central.",
       "implication": {
-        "full": "Once this takes effect, Entities will no longer be generated when Submissions are marked Approved, including {records} we found that has neither been marked Approved nor Rejected. | Once this takes effect, Entities will no longer be generated when Submissions are marked Approved, including {records} we found that have neither been marked Approved nor Rejected.",
-        "records": "{count} record | {count} records"
+        "full": "You currently have {records} not marked Approved nor Rejected.",
```

Comment:
I think this still needs to be pluralized like it was before even though the message doesn't use `{count}`. In some languages, the word "marked" may have to change to match whether "record" is pluralized.